src/sqlfluff/dialects/dialect_oracle.py

```diff
@@ -896,3 +896,21 @@ class UnpivotSegment(BaseSegment):
             Ref("PivotForInGrammar"),
         ),
     )
+
+
+class ObjectReferenceSegment(ansi.ObjectReferenceSegment):
+    """A reference to an object."""
+
+    # Allow whitespace
+    match_grammar: Matchable = Delimited(
+        Ref("SingleIdentifierGrammar"),
+        delimiter=Ref("ObjectReferenceDelimiterGrammar"),
+        terminators=[Ref("ObjectReferenceTerminatorGrammar")],
+        allow_gaps=True,
+    )
+
+
+class ColumnReferenceSegment(ObjectReferenceSegment):
+    """A reference to column, field or alias."""
+
+    type = "column_reference"
```

Comment:
Does this need to be here? Unless we're editing it, we can just rely on the one from the parent dialect. The rest of this looks good.

Reply:
Yes, this need to be there so ColumnReferenceSegment inherit from the new ObjectReferenceSegment.
If I remove this the test fail.

Reply:
Ah yes. I understand now.